install_scripts/templates/kubernetes/migrate.sh

```diff
@@ -0,0 +1,178 @@
+#!/bin/sh
+
+set -e
+
+TMP_DIR=/tmp/migrate-k8s
+LOG_LEVEL=info
+PRIVATE_ADDRESS=
+PUBLIC_ADDRESS=
+AIRGAP=0
+# If all state needed from the native app has been saved we can skip some steps
+HAS_NATIVE_STATE=0
+HAS_KUBERNETES=0
+HAS_APP=0
+
+{% include 'common/cli-script.sh' %}
+{% include 'common/common.sh' %}
+{% include 'common/log.sh' %}
+{% include 'common/system.sh' %}
+
+startNativeScheduler() {
+    logStep "Ensuring native scheduler is running"
+
+    systemctl start replicated
+    systemctl start replicated-operator
+    systemctl start replicated-ui
+
+    # ensure replicatedctl wrapper is for native
+    if ! grep -q "sudo docker exec" "/usr/local/bin/replicatedctl" 2>/dev/null ; then
+        installCliFile "sudo docker exec" "replicated"
+    fi
+
+    waitReplicatedctlReady
+
+    logSuccess "Native scheduler is running"
+}
+
+waitReplicatedctlReady() {
+    for i in {1..30}; do
+        if isReplicatedctlReady; then
+            return 0
+        fi
+        sleep 2
+    done
+    return 1
+}
+
+isReplicatedctlReady() {
+    replicatedctl system status 2>/dev/null | grep -q '"ready"'
```

Comment:
👍 , you can also use `replicated apps`, which is supported by older Replicated versions.

Reply:
I ended up not changing it since 2.33 is required.